src/Microsoft.OData.Client/Materialization/HttpWebResponseMessage.cs

```diff
@@ -159,11 +154,6 @@ public virtual void SetHeader(string headerName, string headerValue)
         /// <returns>Stream from which the response payload can be read.</returns>
         public virtual Stream GetStream()
         {
-#if DEBUG

```

Comment:
Not sure if these should be removed as they could have been used by certain (users) for debugging purposes only.

Reply:
It is not necesary , you should be able to call the function many times. You  can handle stream reads after the call.

Reply:
I think this code here to make sure the "Response Stream" is only used once. You can not use stream multiple time.

---
In reply to: [380228582](https://github.com/OData/odata.net/pull/1664#discussion_r380228582) [](ancestors = 380228582)